docusaurus.config.js

```diff
@@ -1,105 +1,105 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
```

Comment:
Install prettier in a follow-up PR so we can freeze the formatting.

Reply:
will checkout from this branch for that once this PR is approved to avoid any merge conflicts; made a TODO on my end